operator/pom.xml

```diff
@@ -40,13 +40,10 @@
 
     <dependencies>
         <!-- Fabric8 -->
-        <dependency>
-            <groupId>io.fabric8</groupId>
-            <artifactId>crd-generator-api</artifactId>
-        </dependency>
         <dependency>
             <groupId>io.fabric8</groupId>
             <artifactId>crd-generator-apt</artifactId>
```

Comment:
Why do you need this dependency, actually?

Reply:
At the very least it's still transitively bringing in the sundrio dependencies.  I'm not sure historically why crd-generator-apt was needed.

Any thoughts on quarkus-operator-sdk-bundle-generator?